### PR TITLE
Move code to yast-caasp

### PR DESCRIPTION
**Pull request by imobachgs**
This code removes some code that will be moved to the new https://github.com/yast/yast-caasp repository. See https://github.com/yast/yast-caasp/pull/1.

### CODE REVIEW

File: src/lib/installation/system_role_handlers_runner.rb [hunk 1]
```diff
@@ -51,7 +50,7 @@ def finish(role_id)
     #
     # @return [Boolean] True if the file was loaded; false otherwise.
     def require_handler(role_id)
-      filename = "installation/system_role_handlers/#{role_id}_finish"
+      filename = "y2system_role_handlers/#{role_id}_finish"
       require filename
       true
     rescue LoadError
```
Comment (jreidinger) [inline src/lib/installation/system_role_handlers_runner.rb]: in fact this false branch do not return false, but nil

File: doc/system-role-handlers.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    def run
+      # Handler implementation goes here
+    end
+  end
+end
+
+```
+
+The `run` method won't receive any argument and is also not expected to return
+any special value.
```
Comment (lslezak) [inline doc/system-role-handlers.md]: And what about the exceptions? Are they caught (and ignored) or do they buble up and trigger the general YaST exception handler?

Comment (lslezak) [inline doc/system-role-handlers.md]: Just document the behavior...

Comment (imobachgs) [inline doc/system-role-handlers.md]: They just bubble up and get captured by YaST.

**Comment by jreidinger** [comment]
@imobachgs lgtm from my side

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**